zookeeper-client/zookeeper-client-c/CMakeLists.txt

```diff
@@ -290,3 +292,31 @@ if(WANT_CPPUNIT)
     "ZKROOT=${CMAKE_CURRENT_SOURCE_DIR}/../.."
     "CLASSPATH=$CLASSPATH:$CLOVER_HOME/lib/clover*.jar")
 endif()
+
+# set up `make install`.
+# headers end up in /usr/local/include/zookeeper/*.h
+# (with /usr/local being overrideable by using --prefix etc)
+INSTALL(FILES
+    include/zookeeper.h
+    include/proto.h
+    include/config.h
+    include/zookeeper_version.h
+    include/winconfig.h
+    include/zookeeper_log.h
+    include/recordio.h
+    generated/zookeeper.jute.h
+  DESTINATION include/zookeeper/)
+# libzookeeper.a ends up in /usr/local/lib/libzookeeper.a
+INSTALL(TARGETS zookeeper
+  ARCHIVE DESTINATION lib)
+
+if (UNIX)
+  INCLUDE(FindPkgConfig QUIET)
+  if(PKG_CONFIG_FOUND)
+    # generate .pc and install
+    CONFIGURE_FILE("libzookeeper.pc.in" "libzookeeper.pc" @ONLY)
+    INSTALL(FILES libzookeeper.pc
+      DESTINATION lib/pkgconfig)
+  endif()
```

Comment:
I would drop the `INCLUDE` and subsequent `PKG_CONFIG_FOUND` conditional, as generating that small ASCII file does not actually require `pkg-config` at compilation time.  But you prefer to keep it, I'd still suggest switching to `find_package(PkgConfig QUIET)`.